src/component/page/deltakerliste-detaljer/store/koordinator-filter-meny-store-provider.ts

```diff
@@ -99,6 +99,43 @@ export const [ KoordinatorFilterMenyStoreProvider, useKoordinatorFilterMenyStore
 			: filtrerDeltakerePaNavKontor(filtrertPaMedveileder)
 	}
 
+	const updateFiltersToBeValid = (deltakere: TiltakDeltaker[]) => {
```

Comment:
nit: kan godt skrive funksjonsnavnet på norsk

Reply:
Funderer litt på at denne koden er jo egentlig samme som den som sørger for at riktige elementer dukker opp i filtercheckboksene inni filterkomponentene.. Kan være verdt å tenke på om man kan samle implementasjonen sånn at man ikke må gjøre de samme type logikk flere ganger. 

Dette ble såpass spesifikt per filtertype også at jeg er ikke sikker på om det ble noen gevinst i å flytte det ut av komponenten som har ansvar for selve filtreringen.

Reply:
Jeg er enig at logikken er veldig spesifikk. og vi kan unngå litt listeoperasjoner ved å ha de i selve filterkomponenten. Det jeg er redd for er at alt rendres, så vil ev ugyldige filtre bli synlig ved chips et lite halvt sekund, derretter å forsvinne. som et blink, fordi useEffect vil kjøre etter render og så oppdatere store og rendre på nytt. men det er kanskje uansett et problem så lenge vi har logikken i useEffect